newcomers-tutorial.md

```diff
@@ -20,31 +20,31 @@ The path to a successful merge request looks like this:
 
 ## Installing Prerequisites
 
-GHC builds on a plethora of platforms; check [Setting up your system
+GHC builds many platforms; check [Setting up your system
 for building GHC](building/preparation) for detailed instructions for your
 platform of choice.
 
-A minimal list of prerequisites:
+To build GHC, you'll need both a fairly regular set of build tools as well as a Haskell toolchain. The standard tools are:
 
 - **git**
-- **a typical build toolchain**: autoconf, automake, libtool, make, gcc, g++
+- **a standard build toolchain**: autoconf, automake, libtool, make, gcc, g++
 - **python3**
 - **libtinfo**
 - **libgmp**
 - **ncurses**
 - **xz-utils**
 
-Install these as usual. You will need the development packages for any system
-libraries (typically named `something-dev` on a Unix-like OS distribution).
+Install these as usual. You will need the development packages for system
+libraries (usually named like `git-dev`, `python3-dev`, etc. on Unix-like OS distributions).
 
-And a Haskell toolchain:
+The Haskell toolchain has:
```

Comment:
I have explicitly mentioned a haskell toolchain - in the paragraph above both the typical and haskell toolchains.

The wording I'm after here is "includes", I think. The meaning is "here are the tools in the toolchain", but with less words. 

In the introductory paragraph ("You'll need both a typical set of build tools as well as a Haskell toolchain.") I'm attempting to smoothly connect the flow from heading up to individual instructions. I find the lack of links and things like "install these as usual" quite jarring, because they seem less helpful than the previous instructions were (at least, as far as my memory goes).